### ISSUE TITLE
Device gets new ID every time container restarts

### ISSUE THREAD

**Issue by MarkT322** [jlesage/docker-crashplan-pro#88]
My installation has had some crashes, plus I need to restart the NAS (QNAP TS-251) for firmware updates, and every time this happens Crashplan sees it as a new device and asks me if I want to replace an existing device. Is there an environment setting I missed that would address this?

**Comment by jlesage** [comment]
This usually means that you didn't map correctly `/config`.
What is the config of the container?

**Comment by jlesage** [comment]
After a reboot, do you have files in `/share/Container/appdata/crashplan-pro`?  In fact you could check that the content of the directory is the same before and after the reboot.

**Comment by MaizeNBlue** [comment]
I just did both a reboot of the container via the QNAP Console and I also completed the same steps I used to update the container.  Both times this go around there were no issues and the files remained the same in that location.  I also changed the QNAP native "auto-start" flag again, which caused it to reset in the past and that too worked this time with no issues.  I appreciate the quick response and I'll just keep an eye on those files next update and reboot to see if I can find a trend.

Thank you!
